net-p2p/classified-ads/classified-ads-0.10.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PLOCALES="en fi sv da uk"
+PLOCALE_BACKUP="en"
+inherit qmake-utils virtualx vcs-snapshot
+
+COMMIT_ID="cd0652c52f86f6284b793f26e5362bc8fb8a7118"
+DESCRIPTION="Program for displaying classified advertisement items"
+HOMEPAGE="http://katiska.org/classified-ads/"
+SRC_URI="https://github.com/operatornormal/classified-ads/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz \
+	https://github.com/operatornormal/classified-ads/blob/graphics/preprocessed.tar.gz?raw=true \
+		-> classified-ads-graphics-${PV}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="debug doc test"
+
+RDEPEND="dev-libs/openssl:0
+		>=net-libs/libnatpmp-20130911
+		<=net-libs/libnatpmp-20140401-r1
+		>=net-libs/miniupnpc-1.8
+		sys-apps/file
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtnetwork:5[ssl]
+		dev-qt/qtwidgets:5
+		dev-qt/qtsql:5[sqlite]
+		dev-qt/qtmultimedia:5[widgets]
+		dev-qt/qt-mobility[multimedia]
+		dev-qt/qtprintsupport:5
+		media-libs/opus"
+
+DEPEND="${RDEPEND}
+	sys-devel/gettext
+		sys-devel/gdb:0
+	doc? ( app-doc/doxygen[dot] )
+	test? ( dev-libs/libgcrypt:0
+		dev-qt/qttest:5
+		${VIRTUALX_DEPEND} )"
```

Comment:
Yes it does appear redundant since it has not been assigned any further deps, prior to the line listing inherit